proposed/http-message.md

```diff
@@ -1100,7 +1106,8 @@ interface StreamableInterface
      *
      * @see seek()
      * @link http://www.php.net/manual/en/function.fseek.php
-     * @return bool Returns TRUE on success or FALSE on failure.
+     * @return bool Returns TRUE on success.
```

Comment:
Should we then also allow returning FALSE and leave RuntimeException for weird exceptions or not throw at all for this method? Seems weird to me.